lib/msal-node/src/cache/TokenCache.ts

```diff
@@ -116,6 +116,16 @@ export class TokenCache implements ISerializableTokenCache, ITokenCache {
         return this.storage.getCache();
     }
 
+    /**
+     * Gets cache snapshot in CacheKVStore format
+     */
+    getCacheSnapshot(): CacheKVStore {
+        const deserializedPersistentStorage = NodeStorage.generateInMemoryCache(
```

Comment:
can you add a type to this?

Reply:
Why? It's inferred.

Reply:
You're right, but don't you think we should explicitly type everything anyways? I would think the general recommendation is to type everything explicitly - especially for better code clarity, maintainability, etc.

Reply:
In my opinion, explicit typing provides a clearer understanding of the code's intentions.

Reply:
Clarity and readability are not the purpose of types, those should be achieved with descriptive variable/function names and comments, as appropriate. The purpose of types are to provide API contracts and safety. 

Generally speaking, style is enforced by our linter, if there are rules you would like to add or change let's discuss separately from functional changes and, if agreed to by the team, enforce repo wide.

Reply:
Understood, and thanks for the explanation.